virttest/bootstrap.py

```diff
@@ -360,6 +370,15 @@ def _forced_or_detected(forced, detected):
         cfg.write("                            - @%s:\n" % release)
         cfg.write("                                variants:\n")
         cfg.write("                                    - @%s:\n" % arch)
+        cfg.write("variants:\n")
+        cfg.write("    - @Hostvendor:\n")
+        cfg.write("        variants:\n")
+        cfg.write("            - @%s:\n" % vendor)
+        if revision:
+            cfg.write("variants:\n")
+            cfg.write("    - @Hostrevision:\n")
+            cfg.write("        variants:\n")
+            cfg.write("            - @%s:\n" % revision)
```

Comment:
Do you likely want to filter by revision only? If not then the `Hostvendor.$vendor.$revision` should be enough and we should not need to define the `Hostrevision` (therefor `Hostvendor.$vendor.Hostrevision.$revision`).

... still if you think people will want to avoid `Hostversion` and filter only by `Hostrevision`, then it should be exactly as you did here...

Reply:
it is better to be this way, as most of the times we will use Hostversion and very few times a hostrevision...

Reply:
OK, no problem